### PR TITLE
Beautify frontend 1 & 2

### PR DESCRIPTION
**Pull request by mingyuanlee**

### CODE REVIEW

File: src/react/components/StakerComponents/StakerWizard/Steps/Step2-GenerateMnemonic/StepGenerateMnemonic.tsx [hunk 1]
```diff
@@ -11,22 +21,61 @@ import WizardNavigator from "../../WizardNavigator";
...
   setPassword: (password: string) => void;
 }
 
+// const shortenMnemonic = (mnemonic: any) => {
+//   const wordArray = mnemonic.split(" ");
+//   return `${wordArray.slice(0, 3).join(", ")} ... ${wordArray
+//     .slice(-2)
+//     .join(", ")}`;
+// };
+
```
Comment (Nicky-2000) [inline src/react/components/StakerComponents/StakerWizard/Steps/Step2-GenerateMnemonic/StepGenerateMnemonic.tsx:43]: Dead code?

**Comment by Nicky-2000** [comment]
Looks pretty <3 0_0 lolll

File: src/react/components/StakerComponents/StakerWizard/Steps/Step2-GenerateMnemonic/StepGenerateMnemonic.tsx (hunk 1 above)
Comment (dbvora03) [reply src/react/components/StakerComponents/StakerWizard/Steps/Step2-GenerateMnemonic/StepGenerateMnemonic.tsx:43]: @Nicky-2000 ;(

Comment (Nicky-2000) [reply src/react/components/StakerComponents/StakerWizard/Steps/Step2-GenerateMnemonic/StepGenerateMnemonic.tsx:43]: Hahaha sorry 🫣

**Pull request closed without merging**